js/ui/draggable.js

```diff
@@ -70,7 +70,7 @@ class ScrollHelper {
         const that = this;
 
         if(!elements.some(element => that._trySetScrollable(element, mousePosition))) {
-            that._$scrollable = null;
+            that._$scrollableAtMousePosition = null;
```

Comment:
I think that _$scrollableAtPointer or _$scrollableAtCursor name will be shorter and better